src/app/client/src/app/modules/notes/services/notes/notes.service.ts

```diff
@@ -0,0 +1,103 @@
+
+import { LearnerService } from '@sunbird/core';
+import { Injectable, EventEmitter } from '@angular/core';
+import { ConfigService } from '@sunbird/shared';
+import { INotesListData } from '@sunbird/notes';
+
+@Injectable()
+
+export class NotesService {
+  /**
+   * To get url and app config.
+   */
+  config: ConfigService;
+
+  learnerService: LearnerService;
+
+  /**
+   * The constructor
+   * @param {LearnerService} learnerService Reference of LearnerService
+   * @param {ConfigService} config Reference of ConfigService
+   */
+
+  constructor(learnerService: LearnerService, config: ConfigService) {
+    this.config = config;
+    this.learnerService = learnerService;
+  }
+  /**
+   * An event emitter to update notesList after creating a note.
+   */
+  updateNotesListData: EventEmitter<any> = new EventEmitter();
```

Comment:
Rename updateNotesListData to createEventEmitter.
Likewise define updateEventEmitter and deleteEventEmitter.